javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,21 @@
+  
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+//console.log(products.length);
+
+function randerProducts(products){
+    let ulList=document.getElementById("productList");
+
+    for(let i=0;i<products.length;i++){
+    const liElement=document.createElement("li");
+    liElement.innerHTML='<h2>'+products[i].name +'</h2> Price:'+products[i].price+'<br> Rating:'+products[i].rating+'<br> <br>'
```

Comment:
This works, but didn't we learn how to create `h2` tags in javascript via the DOM API?